neon_hana/auth/client_manager.py

```diff
@@ -71,12 +72,12 @@ def check_auth_request(self, client_id: str, username: str,
 
         if not self.rate_limiter.get_all_buckets(f"auth{origin_ip}"):
             self.rate_limiter.add_bucket(f"auth{origin_ip}",
-                                         TokenBucket(replenish_time=30,
-                                                     max_tokens=3))
+                                         TokenBucket(replenish_time=60,
+                                                     max_tokens=self._auth_rpm))
         if not self.rate_limiter.consume(f"auth{origin_ip}"):
             raise HTTPException(status_code=429,
                                 detail=f"Too many auth requests from: "
-                                       f"{origin_ip}. Wait 30 seconds.")
+                                       f"{origin_ip}. Wait 1 minute.")
```

Comment:
FWIW, giving the exact time to wait helps people infer the rate limit, and they can script it more accurately to stay within the window. No matter what the approach is, it's probably best to make sure there's monitoring in place for multiple failed requests inside a time window.

Reply:
Added some logic and update the return to give the time until the rate-limit counter resets.

When we implement some actual auth validation I agree we should also monitor failed requests and impose some stricter limits for bad credentials